README.md

```diff
@@ -74,3 +74,21 @@ To contribute to this repository, please see the [contribution guidelines](./CON
 If you would like to read more about the algorithm used by EiffelVis, please see the [algorithm concept](./frontend/README.md#layout-algorithm).
 
 Copyright © 2022, EiffelVis. EiffelVis is a product by ItJustWorks™.
+
+# Testing
+
+TODO
```

Comment:
Lets not kid ourselves, TODOs are an eternal blight on your legacy

Reply:
🤣 Fait, fair!